lib/util/flattenOperands.js

```diff
@@ -92,23 +88,13 @@ function flattenOperands(root) {
   });
 }
 
-function isPolynomialTerm(node) {
```

Comment:
There's another repo called `math-nodes` which contains `query` and `build` helpers for querying properties of nodes and building new nodes (and trees) respectively.  `isPolynomialTerm` is defined on `query`.